README.md

```diff
@@ -11,8 +11,27 @@ The repository for the backend can be found [here](https://github.com/mozilla/fi
 
 ## Prerequisites
 
-- Node
-- [Yarn](https://www.npmjs.com/package/yarn)
+
+## [Node](https://nodejs.org/en/)
+
+* ## Install node:
```

Comment:
remove the star (`*`), just bold this line, or just remove it because we are in the Developing->Prerequisites-> Node section already

    **Install node**

Reply:
removed install node